src/controllers/user.js

```diff
@@ -0,0 +1,59 @@
+import Model from '../models';
+
+const { Users } = Model;
+
+/**
+* Generic update profile
+
+* @function
+*
+* @param {request} request - express request object
+*
+* @return {Object} - callback that execute the controller
+*
+* @description allows users to update their profile
+*/
+const updateProfile = async (request) => {
+  const {
+    params: { id },
+    body: payload
+  } = request;
+
+  const [numberOfAffectedRows, affectedRows] = await Users.update(
+    { ...payload },
+    { where: { id }, returning: true, raw: true }
+  );
```

Comment:
Please leave a new line understand this block.

Reply:
Done